imx/interface.go

```diff
@@ -47,22 +47,22 @@ type Environment struct {
 type Client struct {
 	Environment          Environment
 	EthClient            *ethclient.Client
-	RegistrationContract *contracts.Registration
-	CoreContract         *contracts.Core
-	api.TradesApi
-	api.OrdersApi
-	api.TransfersApi
-	api.DepositsApi
-	api.WithdrawalsApi
-	api.MintsApi
-	api.AssetsApi
-	api.UsersApi
+	registrationContract *contracts.Registration
+	coreContract         *contracts.Core
+	tradesAPI            api.TradesApi
+	ordersAPI            api.OrdersApi
+	transfersAPI         api.TransfersApi
+	depositsAPI          api.DepositsApi
+	withdrawalsAPI       api.WithdrawalsApi
+	mintsAPI             api.MintsApi
+	assetsAPI            api.AssetsApi
+	usersAPI             api.UsersApi
+	metadataAPI          api.MetadataApi
+	tokensAPI            api.TokensApi
+	balancesAPI          api.BalancesApi
+	projectsAPI          api.ProjectsApi
+	collectionsAPI       api.CollectionsApi
```

Comment:
I'm wondering if we can make the field names more concise.

`assetsAPI` -> `assets`

(but it may have been an established pattern in other SDKs)

Reply:
These are internal parameters. 

We also have Asset struct, can lead to a confusion of list of assets for example.

Reply:
no problem